lib/Controller/ApiController.php

```diff
@@ -1096,6 +1096,39 @@ public function exportSubmissions(string $hash): DataDownloadResponse {
 		$csv = $this->submissionService->getSubmissionsCsv($hash);
 		return new DataDownloadResponse($csv['data'], $csv['fileName'], 'text/csv');
 	}
+	/**
+	 * @NoAdminRequired
+	 * @NoCSRFRequired
+	 *
+	 * Export submissions of a specified Question
+	 *
+	 * @param int $id the question
```

Comment:
```suggestion
	 * @param int $questionId the question
```
This needs to match signature

Reply:
done

Reply:
done